engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -78,6 +81,8 @@ class WorkflowManagerActor(config: Config)
 
   private val donePromise = Promise[Unit]()
 
+  val serviceRegistryActor = context.actorOf(ServiceRegistryActor.props(config), "ServiceRegistoryActor")
```

Comment:
Just a heads up, I moved this to a `ServiceRegistry` trait in my PR so it can be used anywhere (typically the REST endpoint needs it too).